### PR TITLE
task: show author affiliations in author info

### PR DESCRIPTION
**Pull request by ilya-zhuravel**

### CODE REVIEW

File: src/app/containers/manuscript/authors-info-details/index.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { AuthorFormDialog } from 'app/containers/author-form-dialog';
 import { isEqual } from 'lodash';
 import { stringifyEditorState } from 'app/utils/view.utils';
 import { OrcidIcon } from 'app/assets/icons';
+import { getAffiliationDisplayName } from 'app/models/affiliation';
```
Comment (NuclearRedeye) [inline src/app/containers/manuscript/authors-info-details/index.tsx:19]: I might not have been involved in all the discussion, but in the [design](https://projects.invisionapp.com/d/main#/console/19776007/414680496/preview) the text is...

```
Fred Atherden is affiliated with the Production Department, eLife Sciences, Cambridge
```
Also, the authors name shouldn't be bold?

Comment (ilya-zhuravel) [reply src/app/containers/manuscript/authors-info-details/index.tsx:19]: What you are referring to is actually Bio. The affiliations are listed after the email.

Comment (NuclearRedeye) [reply src/app/containers/manuscript/authors-info-details/index.tsx:19]: 🤦

Comment (NuclearRedeye) [reply src/app/containers/manuscript/authors-info-details/index.tsx:19]: In my defence, it doesn't help that the text for the bio is the same as the affiliation! But yeah I see it now...

**Pull request merged**